### PR TITLE
fix bag

### PR DESCRIPTION
**Pull request by fantomkis**

### CODE REVIEW

File: src/app/number/number.component.html [hunk 1]
```diff
@@ -1,6 +1,23 @@
-<p>Клікни на заголовок і число збільшиться</p>
+<p [ngClass]="{'green': isNoun, 'red': !isNoun}">Click on the title and the number will increase
+</p>
 
+<h3>{{ counter }}</h3>
 
-<h3>{{counter}}</h3>
+<!-- <div *ngif="counter else elseblock">
```
Comment (AlexKovalchuk) [inline src/app/number/number.component.html]: Remove comments

File: src/app/number/number.component.ts [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
   constructor() { }
 
   ngOnInit(): void {
   }
 
+  ngOnChanges(changes: SimpleChanges): void {
+    this.isNoun = changes['counter'].currentValue % 2 === 0;
+  }
+
   // onClick(){
```
Comment (AlexKovalchuk) [inline src/app/number/number.component.ts]: Click is not rerendering the page?

File: src/app/number/number.component.html (hunk 1 above)
Comment (fantomkis) [reply src/app/number/number.component.html]: +

Comment (fantomkis) [reply src/app/number/number.component.html]: done

**Pull request merged**